src/solvers/z3/z3_conv.cpp

```diff
@@ -6,7 +6,12 @@
  \*******************************************************************/
 
 #include <cassert>
+#include <esbmc/bmc.h>
 #include <z3_conv.h>
+#include <fmt/format.h>
+#include <fstream>
+#include <stdio.h>
+#include <stdlib.h>
 #include <util/message/default_message.h>
```

Comment:
please, cleanup these includes. I probably only need fstream here.

Reply:
Made this change